node/src/Transaction.ts

```diff
@@ -1093,6 +1094,21 @@ export class BaseTransaction<T extends BaseTransaction<T>> {
     public time(): T {
         return this.addAndReturn(createTime());
     }
+
+    /**
+     * Blocking list pop primitive.
+     * Pop an element from the tail of the first list that is non-empty,
+     * with the given keys being checked in the order that they are given.
+     * Blocks the connection when there are no elements to pop from any of the given lists.
+     * See https://redis.io/commands/brpop/ for more details.
+     *
+     * @param keys - The `keys` of the lists to pop from.
+     * @param timeout - The `timeout` in seconds.
+     * @returns - `Array` of the `key` that elements were poped from, and the popped elements, or null if timeout is reached.
```

Comment:
We don't use @returns in transactions but `Command Response:` (see below)

Reply:
I'll fix, but its not consistent throw the file, a lot of the function use `returns`. Need to be aligned in another PR.

Reply:
Can you please open a fix for that?

Reply:
What is a fix? like issue, or you ask to fix?